src/modules/Packages.rb

```diff
@@ -749,9 +750,18 @@ def product_update_summary(products)
 
           re = HTML.Colorize(msg, "red")
         else
+          selected_products = Y2Packager::Product.with_status(:selected)
+          # get the human readable product names from libzypp data
+          new_products = obsolete.map do |p|
+            new_product = selected_products.find { |selected_prod| selected_prod.name == p }
+            # fallback to the internal product name if not found (should never happen™,
+            # ProductUpgrade.will_be_obsoleted_by should return only the selected products)
+            new_product ? new_product.display_name : p
+          end
```

Comment:
hmm, I know it is something for SP1, but I do not like much adding to method with 57 lines another 10 lines. Is possible to safely move it to own method?